app/services/message_employee_matcher.rb

```diff
@@ -12,22 +12,24 @@ def run
   attr_reader :message
 
   def retrieve_matching_employees
-    Employee.where(started_on: day_count.days.ago)
-      .select { |employee| match_time(employee.time_zone) && message_not_already_sent?(employee) }
+    Employee.where(started_on: day_count.days.ago).select do |employee|
+      time_to_send_message?(employee.time_zone) && message_not_already_sent?(employee)
+    end
   end
 
   def day_count
     message.days_after_start
   end
 
-  def match_time(time_zone)
+  def time_to_send_message?(time_zone)
     employee_current_time = Time.current.in_time_zone(time_zone)
 
-    if employee_current_time.day == Time.current.day
-      employee_current_time_value = employee_current_time.strftime("%H%M").to_i
-      message_time_value = message.time_of_day.strftime("%H%M").to_i
-      employee_current_time_value >= message_time_value
-    end
+    return false if employee_current_time.day != Time.current.day
```

Comment:
As I mentioned in my commit message, I made this change for reasons beyond readability. With the previous version of the method, when the condition fails, the method will return `nil`. With my change, it will return `false`. I believe a predicate method should return `true` or `false`, not `true` or `nil`. 

Apparently, this is a common debate in the Ruby world with opinions on both sides, but I tend to agree with the viewpoint that a method should represent its intent. See this article for example: http://pragmati.st/2012/03/24/the-elements-of-ruby-style-predicate-methods/

If the intent is to return whether or not it is time to send the message, I think it's clearer to return `false` as opposed to `nil`, and doing that via an early return is better IMO than adding an `else false` at the end.